gubernator/templates/node_404.html

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
+<title>Missing kubelet.log file</title>
+Unable to load kubelet.log file from <a href="https://console.cloud.google.com/storage/browser{{build_dir}}">gs:/{{build_dir}}</a> for {{pod_name}}
```

Comment:
nit: I would change `load` to `find`, since the problem is probably that the pod_name was not matched in the kubelet file.